apps/ui/src/views/Proposal.vue

```diff
@@ -261,181 +259,187 @@ watchEffect(() => {
         </UiScrollerHorizontal>
         <router-view :proposal="proposal" />
       </div>
-      <Affix
+
+      <UiResizableHorizontal
+        id="proposal-sidebar"
+        :default="340"
+        :max="400"
```

Comment:
Is there a reason to limit it to a max of 400? 

some choices are still not visible ([this proposal reported by cvx](http://127.0.0.1:8080/#/s:cvx.eth/proposal/0xa62fc6f66fc6fd4b3cbbbfb857ec2cc3c95fe9566cb2e1294e9787a46bea0be1))
<img width="390" alt="image" src="https://github.com/user-attachments/assets/6f5aca47-d208-4b7e-bcd1-a9ee3366322f" />

Reply:
maybe we should make these clickable on mobile to read the full choice text

Reply:
400px is the max width set by the spec.

We could increase it, but there will be a limit before we need to refactor the code, to handle case where the sidebar is bigger than page size.

Reply:
apart from that all good we can merge and look at increasing it in future

Reply:
We can set the max to 440px, I wouldn't make this clickable